aspnetcore/fundamentals/servers/kestrel/http3.md

```diff
@@ -48,22 +44,13 @@ HTTP/3 isn't currently supported on macOS and may be available in a future relea
 
 ## Getting started
 
-HTTP/3 is not enabled by default. Add configuration to `Program.cs` to enable HTTP/3.
-
-:::code language="csharp" source="samples/6.x/KestrelSample/Snippets/Program.cs" id="snippet_Http3" highlight="7-8":::
-
-The preceding code configures port 5001 to:
-
-* Use HTTP/3 alongside HTTP/1.1 and HTTP/2 by specifying `HttpProtocols.Http1AndHttp2AndHttp3`.
-* Enable HTTPS with `UseHttps`. HTTP/3 requires HTTPS.
-
-Because not all routers, firewalls, and proxies properly support HTTP/3, HTTP/3 should be configured together with HTTP/1.1 and HTTP/2. This can be done by specifying [`HttpProtocols.Http1AndHttp2AndHttp3`](xref:Microsoft.AspNetCore.Server.Kestrel.Core.HttpProtocols.Http1AndHttp2AndHttp3) as an endpoint's supported protocols.
+HTTP/3 is enabled by default. Because not all routers, firewalls, and proxies properly support HTTP/3, by default HTTP/3 is configured together with HTTP/1.1 and HTTP/2.
 
-For more information, see <xref:fundamentals/servers/kestrel/endpoints>.
+For more information, see <xref:fundamentals/servers/kestrel/endpoints#listenoptionsprotocols>.
 
 ## Alt-svc
 
-HTTP/3 is discovered as an upgrade from HTTP/1.1 or HTTP/2 via the [`alt-svc`](https://developer.mozilla.org/docs/Web/HTTP/Headers/Alt-Svc) header. That means the first request will normally use HTTP/1.1 or HTTP/2 before switching to HTTP/3. Kestrel automatically adds the `alt-svc` header if HTTP/3 is enabled.
```

Comment:
"if HTTP/3 is enabled" is still relevant. If you turn off HTTP/3 then alt-svc is also turned off.